### PR TITLE
Orca: update orca-in-5-minutes to tf2 estimator.

### PR DESCRIPTION
**Pull request by lalalapotter**
## Description
This PR is to update orca-in-5-minutes document to use tf2 estimator.

### CODE REVIEW

File: docs/readthedocs/source/doc/Orca/Overview/orca.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ Most AI projects start with a Python notebook running on a single laptop; howeve
 
 First of all, follow the steps [here](install.md#to-use-basic-orca-features) to install Orca in your environment.
 
-This section uses TensorFlow 1.15, and you should also install TensorFlow before running this example:
+This section uses TensorFlow 2.9.1, and you should also install TensorFlow before running this example:
 ```bash
-pip install tensorflow==1.15
+pip install tensorflow==2.9.1
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/Overview/orca.md:15]: maybe do not restrict tf version?

File: docs/readthedocs/source/doc/Orca/Overview/orca.md [hunk 2]
```diff
@@ -27,33 +27,53 @@ sc = init_orca_context(cluster_mode="local", cores=4, memory="10g", num_nodes=1)
...
-predictions = keras.layers.Dense(2, activation='softmax')(feat)
-model = keras.models.Model(inputs=[user, item], outputs=predictions)
-model.compile(optimizer='rmsprop',
-              loss='sparse_categorical_crossentropy',
-              metrics=['accuracy'])
-
-est = Estimator.from_keras(keras_model=model)
+from bigdl.orca.learn.tf2.estimator import Estimator
+
+def model_creator(config):
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/Overview/orca.md:52]: put something in config?

File: docs/readthedocs/source/doc/Orca/Overview/orca.md [hunk 3]
```diff
@@ -27,33 +27,53 @@ sc = init_orca_context(cluster_mode="local", cores=4, memory="10g", num_nodes=1)
...
+  mlp_embed_user = keras.layers.Embedding(input_dim=num_users, output_dim=8,
+                               input_length=1)(user_input)
+  mlp_embed_item = keras.layers.Embedding(input_dim=num_items, output_dim=8,
+                               input_length=1)(item_input)
+
+  user_latent = keras.layers.Flatten()(mlp_embed_user)
+  item_latent = keras.layers.Flatten()(mlp_embed_item)
+
+  mlp_latent = keras.layers.concatenate([user_latent, item_latent], axis=1)
+  predictions = keras.layers.Dense(2, activation='softmax')(mlp_latent)
```
Comment (hkvision) [inline docs/readthedocs/source/doc/Orca/Overview/orca.md:64]: 2-class change to sigmoid?

**Review by hkvision — APPROVED** [review]
